setup.ts

```diff
@@ -118,6 +118,40 @@ async function redisConfiguration(): Promise<void> {
   }
 }
 
+//LAST_RESORT_SUPERADMIN_EMAIL prompt
+async function askForSuperAdminEmail(): Promise<string> {
+  const { email } = await inquirer.prompt([
+    {
+      type: "input",
+      name: "email",
+      message:
+        "Enter the email which you wish to assign as the super admin of last resort:",
```

Comment:
In all cases use Super Admin in the prompts and messaging. That is what we have in all our documentation and clearly stated we are referring to a role.

Chang in the other locations too

Reply:
I've committed the changes; if they don't meet your expectations, could you please provide guidance by suggesting a replacement sentence?